src/ui_fsmenu/launch_game.cc

```diff
@@ -251,3 +333,7 @@ void FullscreenMenuLaunchGame::clicked_ok() {
 void FullscreenMenuLaunchGame::clicked_back() {
 	NEVER_HERE();
 }
+
+bool FullscreenMenuLaunchGame::clicked_select_map() {
+	NEVER_HERE();
```

Comment:
Instead if having several functions that just do `NEVER_HERE` change their declarations to `some_function() = 0;`

Reply:
done